admin_manual/configuration_files/encryption_details.rst

```diff
@@ -26,7 +26,7 @@ This is the default encryption mode in Nextcloud. With master key encryption ena
 Key type: public sharing key
 ----------------------------
 
-The public sharing key is used to secure files that have been publicly shared. The public sharing key is protected by a password that can be generated by the server administrator. The advantage of the public sharing key is that it is independent of the selected encryption mode so that Nextcloud is able to provide publicly shared files to outside parties.
+The public sharing key is used to secure files that have been publicly shared. The public sharing key is protected by a password that is automatically generated once encryption is enabled. The advantage of the public sharing key is that it is independent of the selected encryption mode so that Nextcloud is able to provide publicly shared files to outside parties.
```

Comment:
Looking at the code, the sharing key is not protected by a password at all.

The master key however, is automatically generated. Paragraph just above.